opensrp-chw-core/src/test/java/org/smartregister/chw/core/repository/MalariaRegisterRepositoryTest.java

```diff
@@ -0,0 +1,51 @@
+package org.smartregister.chw.core.repository;
+
+import net.sqlcipher.database.SQLiteDatabase;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.smartregister.chw.core.BaseRobolectricTest;
+
+public class MalariaRegisterRepositoryTest extends BaseRobolectricTest {
+    @Mock
+    private MalariaRegisterRepository malariaRegisterRepository;
+
+    @Mock
+    private SQLiteDatabase database;
+
+    @Captor
+    ArgumentCaptor<String> tableNameCaptor;
+    @Captor
+    ArgumentCaptor<String[]> tableColumnsCaptor;
+    @Captor
+    ArgumentCaptor<String> selectionCaptor;
+    @Captor
+    ArgumentCaptor<String[]> selectionArgsCaptor;
+    @Captor
+    ArgumentCaptor<String> groupByCaptor;
+    @Captor
+    ArgumentCaptor<String> havingCaptor;
+    @Captor
+    ArgumentCaptor<String> orderByCaptor;
+
+
+    @Before
+    public void setUp() throws Exception {
+        malariaRegisterRepository = Mockito.spy(new MalariaRegisterRepository());
+        Mockito.doReturn(database).when(malariaRegisterRepository).getReadableDatabase();
+    }
+
+    @Test
+    public void testDatabaseQueryIsCreatedWithTheCorrectArgumentInGetFamilyNamePhone(){
+        String baseEntityId = "4b3e6408-0549-470a-b24a-82ac71180a30";
```

Comment:
If  this test covers the following lines https://github.com/opensrp/opensrp-client-chw-core/blob/master/opensrp-chw-core/src/main/java/org/smartregister/chw/core/repository/MalariaRegisterRepository.java#L42-L51, add an assertion for this also since this will not longer be covered having been already marked as code covered by tests but not actually tested